Lib/unittest/mock.py

```diff
@@ -402,6 +403,10 @@ def __init__(self, /, *args, **kwargs):
 class NonCallableMock(Base):
     """A non-callable version of `Mock`"""
 
+    # Store a mutex as a class attribute to protect concurrent accesses to
```

Comment:
Maybe explain briefly, in the comments, why we chose to use a class attribute instead of an instance-level one.

Reply:
I edited the comment!